projects/Time to load Website/time_to_load_website.py

```diff
@@ -0,0 +1,27 @@
+from urllib.request import urlopen
+import time
+
+#Storing user defined URl
+URL = input("Enter the url whose loading time you want to check: ")
+
+#Open the url as entered by the user and checking protocols
+if ("https" or "http") in URL:
+    open_this_url = urlopen(URL)
+
+else:
+    open_this_url = urlopen("https://" + URL)
+
+#Time stamp before the reading of url starts
+start_time = time.time()
```

Comment:
use inline comments for single statement
```suggestion
start_time = time.time()  # Time stamp before the reading of url starts
```